data/mn/retired/Bill-Weber-f839da78-1e9c-4e4b-b1d7-d203970ec6b2.yml

```diff
@@ -8,16 +8,13 @@ image: https://www.senate.mn/img/member_thumbnails/22Weber.jpg
 party:
 - name: Republican
 roles:
-- type: upper
+- end_date: '2023-01-03'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:mn/government
```

Comment:
@bfossen-ce this will need to be moved to legislature.

Reply:
Updated